go-controller/pkg/ovn/egressip_test.go

```diff
@@ -480,11 +488,6 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 						Items: []v1.Node{node1, node2},
 					})
 
-				fakeOvn.fakeExec.AddFakeCmdsNoOutputNoError(
-					[]string{
-						fmt.Sprintf("ovn-nbctl --timeout=15 set logical_switch_port etor-GR_node1 options:nat-addresses=router"),
-					},
-				)
```

Comment:
Same comment as above, please check that the router option is added/removed on the nodes expected by the original test.

Reply:
done